hazelcast/src/test/java/com/hazelcast/internal/nio/ascii/TextProtocolsConfigTest.java

```diff
@@ -271,21 +172,12 @@ public void testRestPropertyDisabled() throws Exception {
 
     @Test
     public void testAllRestPropertiesEnabled() throws Exception {
-        Config config = new Config()
-                .setProperty(GroupProperty.REST_ENABLED.getName(), "true")
-                .setProperty(GroupProperty.HTTP_HEALTHCHECK_ENABLED.getName(), "true")
-                .setProperty(GroupProperty.MEMCACHE_ENABLED.getName(), "true")
-                ;
+        Config config = new Config();
+        config.getNetworkConfig().getMemcacheProtocolConfig().setEnabled(true);
+        config.getNetworkConfig().getRestApiConfig().setEnabled(true);
```

Comment:
This test needs the `DATA` endpoint group enabled. This is the last remaining test failure; it would be great if you can also squash your commits so once the PR builder passes we can merge this PR.